ts/state/ducks/conversations.ts

```diff
@@ -3909,29 +3911,46 @@ function saveAttachments(
       }
     }
 
-    // TODO shouldn't we throw some kind of error here? Original code also doesn't do anything, but...
-    if (fullPath == null)
+    if (fullPath == null) {
       throw new Error('saveAttachments: Returned path to attachment is null!');
+    }
 
     dispatch({
       type: SHOW_TOAST,
       payload: {
-        toastType: ToastType.FileSaved, // TODO we need a new toast here
+        toastType: ToastType.FileSaved,
         parameters: {
-          fullPath: fullPath,
+          countOfFiles: attachments.length,
+          fullPath,
         },
       },
     });
   };
 }
 
+function showAttachmentDownloadStillInProgressToast(
+  count: number
+): ShowToastActionType {
+  log.info(
+    "showAttachmentDownloadStillInProgressToast download still in progress toast for an attachment that the user want's to save"
```

Comment:
Maybe shorten to `showAttachmentDownloadStillInProgressToast: ${count} still-pending attachments`?